packages/web-components/fast-components/src/accordion-item/accordion-item.definition.ts

```diff
@@ -11,21 +11,23 @@ export const fastAccordionItemDefinition: WebComponentDefinition = {
             attributes: [
                 {
                     name: "heading-level",
-                    description: "The heading level attribute",
+                    title: "Heading level",
+                    description: "The aria-level value (1-6) for the item heading",
```

Comment:
```suggestion
                    title: "Level",
                    description: "The aria-level value (1-6) for the item heading denoting it's importance",
```

Reply:
Is it more "hierarchy" instead of "importance"? Aria is always nested sequentially, right?

Reply:
Good feedback - can we agree on "representing its semantic hierarchy"? I think that threads the needle well